TrustKit/Pinning/ssl_pin_verifier.m

```diff
@@ -137,3 +140,38 @@ TSKTrustEvaluationResult verifyPublicKeyPin(SecTrustRef serverTrust, NSString *s
     CFRelease(serverTrust);
     return TSKTrustEvaluationFailedNoMatchingPin;
 }
+
+TSKTrustEvaluationResult verifyCertificatePublicKeyPin(SecCertificateRef certificate,
+                                                       NSString *serverHostname,
+                                                       NSArray<NSNumber *> *supportedAlgorithms,
+                                                       NSSet<NSData *> *knownPins,
+                                                       TSKSPKIHashCache *hashCache)
+{
+    CFStringRef certificateSubject = SecCertificateCopySubjectSummary(certificate);
```

Comment:
Pretty wasteful to extract `certificateSubject` in production just in case we need it for debugging. But I can respect that it was already there in the previous version.